shared/protocol/src/main/java/io/pravega/shared/NameUtils.java

```diff
@@ -40,6 +40,9 @@
     // The Prefix which is used when creating internally used pravega streams.
     public static final String INTERNAL_SCOPE_PREFIX = INTERNAL_SCOPE_NAME + "/";
 
+    // The scope name which has to be used when creating internally used pravega streams.
+    public static final String DELETING_SCOPES_TABLE = "_system/deletingScopes";
```

Comment:
any specific reason why this is added in shared protocol lib? If not let's keep it internal to the controller.

Reply:
This was mainly added as this class is being used for the same purpose across. But a valid point, I'll move it to controller specific class. Thank you